### PR TITLE
add description Fee Gathering BMC message trigger.

### PR DESCRIPTION
**Pull request by leeheonseung**

### CODE REVIEW

File: IIPS/iip-35.md [hunk 1]
```diff
@@ -91,4 +91,4 @@ TBD
 
 ## Copyright
 
-Copyright and related rights waived via [CC0](https://creativecommons.org/publicdomain/zero/1.0/).
```
Comment (sink772) [inline IIPS/iip-35.md:94]: Please no not touch this last line that is not related to your changes. It should have newline at EOF.

**Review by sink772 — CHANGES_REQUESTED** [review]

File: IIPS/iip-35.md (hunk 1 above)
Comment (leeheonseung) [reply IIPS/iip-35.md:94]: I checked.

**Review by sink772 — APPROVED** [review]
